src/main/java/zd/zero/waifu/motivator/plugin/personality/core/IdlePersonalityCore.kt

```diff
@@ -27,9 +28,17 @@ class IdlePersonalityCore : PersonalityCore {
                             project,
                             asset,
                             motivationEvent.alertConfigurationSupplier()
-                    ).setListener {
-                        displayedProjects.remove(project)
-                    }.motivate()
+                    ).setListener(
+                        object : MotivationListener {
+                            override fun onDismissed() {
+                                displayedProjects.remove(project)
+                            }
+
+                            override fun onNotDisplayed() {
+                                displayedProjects.remove(project)
+                            }
```

Comment:
What's the purpose of `onNotDisplayed()` if both of them are doing the same thing?

Reply:
🤔 Hmm, the current use case for this class is that it needs to know when to remove it, so it is the same. However, that might change if it is used someplace else. It isn't right now, so there isn't a need to make the distinction. I guess the name of YAGNI (you ain't gonna need it), i'll consolidate it. Naming stuff is a pain in the rear.

Reply:
> Naming stuff is a pain in the rear.

Couldn't agree moar!